optuna/visualization/_contour.py

```diff
@@ -298,24 +298,17 @@ def _generate_contour_subplot(
             )
         z[y_i][x_i] = value
 
-    # TODO(Yanase): Use reversescale argument to reverse colorscale if Plotly's bug is fixed.
-    # If contours_coloring='heatmap' is specified, reversescale argument of go.Contour does not
-    # work correctly. See https://github.com/pfnet/optuna/issues/606.
-    colorscale = plotly.colors.PLOTLY_SCALES["Blues"]
-    if direction == StudyDirection.MAXIMIZE:
-        colorscale = [[1 - t[0], t[1]] for t in colorscale]
-        colorscale.reverse()
-
     contour = go.Contour(
         x=x_indices,
         y=y_indices,
         z=z,
         colorbar={"title": target_name},
-        colorscale=colorscale,
+        colorscale="blues",
```

Comment:
```suggestion
        colorscale=plotly.colors.PLOTLY_SCALES["Blues"],
```

Now, we use `PLOTLY_SCALES["Blues"]` as a colormap.

Reply:
Changes applied @himkt